src/core/ext/filters/client_channel/resolver/dns/c_ares/grpc_ares_wrapper.cc

```diff
@@ -96,12 +101,38 @@ static void grpc_ares_request_ref(grpc_ares_request* r) {
   gpr_ref(&r->pending_queries);
 }
 
+static void log_address_sorting_list(grpc_lb_addresses* lb_addrs,
+                                     const char* input_output_str) {
+  for (size_t i = 0; i < lb_addrs->num_addresses; i++) {
+    char* addr_str;
+    if (grpc_sockaddr_to_string(&addr_str, &lb_addrs->addresses[i].address,
+                                true)) {
+      gpr_log(GPR_INFO,
+              "C-ares sockaddr address sorting %s index: %" PRIdPTR
```

Comment:
Keep it simple:

    "c-ares address_sorting: %s[%zu]=%s"

(Note that this capitalization style is used by https://c-ares.haxx.se/)

Is there precedent for using `PRIdPTR` with `size_t`?  It seems to be designed for `intptr_t`.

Reply:
Unfortunately I can't use the `%zu` formatters only because they're not supported by mingw, which is needed to build ruby on windows

Reply:
Okay, but you can still simplify the rest of the text.

Reply:
Fixed, simplified text in latest update

Reply:
I don't understand the choice of `input:[0]` over `input[0]` here, but meh.

Reply:
whoops that was a mistaken update, fixed